gocourse03/drugs/main.go

```diff
@@ -0,0 +1,56 @@
+/*
+Треба відсортувати ящик ліків за датою виробництва.
+Застарілі
+Нові
+яким півроку
+*/
+package main
+
+import (
+	"fmt"
+	"github.com/brianvoe/gofakeit/v7"
+	"time"
+)
+
+type Drugs struct {
+	Name         string
+	Manufactured time.Time
+}
+
+func main() {
+	fake := gofakeit.New(0)
+	dateRangeAt := time.Now().AddDate(-2, 0, 0)
+	dateRangeTo := time.Now().AddDate(0, 0, 1)
+	boxOfDrugs := []Drugs{
+		{"Acetaminophen", fake.DateRange(dateRangeAt, dateRangeTo)},
+		{"Cyclobenzaprine", fake.DateRange(dateRangeAt, dateRangeTo)},
+		{"Lofexidine", fake.DateRange(dateRangeAt, dateRangeTo)},
+		{"Pantoprazole", fake.DateRange(dateRangeAt, dateRangeTo)},
+		{"Methadone", fake.DateRange(dateRangeAt, dateRangeTo)},
+		{"Farxiga", fake.DateRange(dateRangeAt, dateRangeTo)},
+		{"Brilinta", fake.DateRange(dateRangeAt, dateRangeTo)},
+	}
+
+	var oldDrugs = make([]Drugs, 0)
+	var newDrugs = make([]Drugs, 0)
+	var sixMonthDrugs = make([]Drugs, 0)
```

Comment:
Здається, я вже коментив про це спрощення:
```suggestion
	var oldDrugs []Drugs
	var newDrugs []Drugs
	var sixMonthDrugs []Drugs
```

В реальному коді зустрічається більше мій варіант, ніж `make` з `0`.